packages/client-core/src/user/menus/SettingsMenu.tsx

```diff
@@ -0,0 +1,386 @@
+            logger.info({ event_name: `set_microphone_volume`, event_value: value })
+          }}
+          onRelease={() => {}}
+          label={
+            (
+              <div className="w-32 whitespace-nowrap text-[#080808]">{t('user:usermenu.setting.lbl-microphone')}</div>
+            ) as any
+          }
+        />
+      </div>
+      <div className="mx-auto my-1 w-3/4">
+        <Slider
+          max={1}
+          min={0}
+          step={0.01}
+          value={audioState.mediaStreamVolume.value}
+          onChange={(value: number) => {
+            audioState.mediaStreamVolume.set(value)
+            logger.info({ event_name: `set_user_volume`, event_value: value })
+          }}
+          onRelease={() => {}}
+          label={
+            (
+              <div className="w-32 whitespace-nowrap text-[#080808]">
+                {t('user:usermenu.setting.lbl-media-instance')}
+              </div>
+            ) as any
+          }
+        />
+      </div>
+      <div className="mx-auto my-1 w-3/4">
+        <Slider
+          max={1}
+          min={0}
+          step={0.01}
+          value={audioState.notificationVolume.value}
+          onChange={(value: number) => {
+            audioState.notificationVolume.set(value)
+            logger.info({ event_name: `set_notification_volume`, event_value: value })
+          }}
+          onRelease={() => {}}
+          label={
+            (
+              <div className="w-32 whitespace-nowrap text-[#080808]">{t('user:usermenu.setting.lbl-notification')}</div>
+            ) as any
+          }
+        />
+      </div>
+      <div className="mx-auto my-1 w-3/4">
+        <Slider
+          max={1}
+          min={0}
+          step={0.01}
+          value={audioState.soundEffectsVolume.value}
+          onChange={(value: number) => {
+            audioState.soundEffectsVolume.set(value)
+            logger.info({ event_name: `set_scene_volume`, event_value: value })
+          }}
+          onRelease={() => {}}
+          label={
+            (
+              <div className="w-32 whitespace-nowrap text-[#080808]">{t('user:usermenu.setting.lbl-sound-effect')}</div>
+            ) as any
+          }
+        />
+      </div>
+      <div className="mx-auto my-1 w-3/4">
+        <Slider
+          max={1}
+          min={0}
+          step={0.01}
+          value={audioState.backgroundMusicVolume.value}
+          onChange={(value: number) => {
+            audioState.backgroundMusicVolume.set(value)
+            logger.info({ event_name: `set_music_volume`, event_value: value })
+          }}
+          onRelease={() => {}}
+          label={
+            (
+              <div className="w-32 whitespace-nowrap text-[#080808]">
+                {t('user:usermenu.setting.lbl-background-music-volume')}
+              </div>
+            ) as any
+          }
+        />
+      </div>
+    </div>
+  )
+}
+
+function GraphicsTab() {
```

Comment:
I can see the profile menu behind the graphics settings. we should probably hide it or make both dialogs the same size.
![Screenshot 2025-01-29 at 2 40 37 p m](https://github.com/user-attachments/assets/4810e9ed-48bc-4aab-b2b6-a2f46ef59bad)

Reply:
Fixed